input/otlp/traces.go

```diff
@@ -670,14 +791,19 @@ func TranslateSpan(spanKind ptrace.SpanKind, attributes pcommon.Map, event *mode
 	case httpSpan:
 		if httpResponse.StatusCode > 0 {
 			if event.Event.Outcome == outcomeUnknown {
-				event.Event.Outcome = clientHTTPStatusCodeOutcome(httpResponse.StatusCode)
+				event.Event.Outcome = clientHTTPStatusCodeOutcome(int(httpResponse.StatusCode))
 			}
 		}
 		event.Span.Type = "external"
 		subtype := "http"
 		event.Span.Subtype = subtype
-		event.HTTP = http
-		event.URL.Original = httpURL
+		if !proto.Equal(&http, &modelpb.HTTP{}) {
+			event.Http = &http
+		}
```

Comment:
same as above, shouldn't it be expected to be non-empty? can we drop the proto.Equal?

Reply:
Removed :+1: